rascaline/examples/compute-soap.rs

```diff
@@ -17,7 +17,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         "max_angular": 4,
         "atomic_gaussian_width": 0.3,
         "center_atom_weight": 1.0,
-        "gradients": true,
```

Comment:
Now, this example does not compute the gradients. It never used them but in that sense it is different. Probably leaving the computations in the example (even without using the gradients) would show a user how to do it.